research/sentiment_analysis/README.md

```diff
@@ -0,0 +1,23 @@
+# Sentiment Analysis
+## Overview
+This is an implementation of the Sentiment Analysis model as described in the [this](https://arxiv.org/abs/1412.1058) paper. The implementation is with the reference to [paddle version](https://github.com/mlperf/reference/tree/master/sentiment_analysis/paddle).
+
+The model makes use of concatenation of two CNN layers with different kernel sizes. Dropout and batch normalization layers are used to prevent over fitting.
+
+## Dataset
+The [keras](https://keras.io)'s [IMDB Movie reviews sentiment classification](https://keras.io/datasets/#imdb-movie-reviews-sentiment-classification) dataset is used.
```

Comment:
It's good if you can add more details about the dataset, like the size, format, or how long it takes to download.

Reply:
ack (I will also note that the the download is prompt)